devel/centos9_build_setup.yml

```diff
@@ -0,0 +1,76 @@
+---
+- hosts: cluster
+  become: yes
+  tasks:
+
+    - name: Enable CRB repo
+      command: yum config-manager --set-enabled crb
+
+    - name: Install required rpms
+      yum:
+        name:
+          - git
+          - gdb
+          - gcc
+          - python3
+          - perl
+          - automake
+          - autoconf
+          - libtool
+          - flex
+          - bison
+          - python3-pip
+          - openssl-devel
+          - libxml2-devel
+          - python3-devel
+          - libaio-devel
+          - libibverbs-devel
+          - librdmacm-devel
+          - readline-devel
+          - lvm2-devel
+          - glib2-devel
+          - userspace-rcu-devel
+          - libcmocka-devel
+          - libacl-devel
+          - sqlite-devel
+          - fuse-devel
+          - cups-devel
+          - dbus-devel
+          - docbook-style-xsl
+          - libarchive-devel
+          - libcap-devel
+          - libldb-devel
+          - libtalloc-devel
+          - libtdb-devel
+          - libtevent-devel
+          - libxslt
+          - openldap-devel
+          - pam-devel
+          - perl-ExtUtils-MakeMaker
+          - perl-Parse-Yapp
+          - perl-Test-Simple
+          - popt-devel
+          - python-tdb
+          - python-tevent
+          - gnutls-devel
+          - libtirpc
+          - libtirpc-devel
+          - libnsl2-devel
+          - rpcgen
+          - jansson-devel
+        state: latest
+
+    - name: Install python modules
+      pip:
+        executable: /usr/bin/pip3
+        name:
+          - iso8601
+          - pyasn1
+          - cryptography==3.3.1
+
+    - name: Install gluster devel packages
+      yum:
+        name:
+          - glusterfs-api-devel
+          - glusterfs-devel
+        state: present
```

Comment:
True. However the contents of *devel* directory are not integrated with all the backend logic. It also doesn't respect the `use_distro` extra variable and requires to explicitly choose the OS release (which may not match the installed machines). IMO we should move this to the main playbooks and install it if required, but this should be done in another PR.

Also, shouldn't we install the debug packages instead of the devel ones ? do we really plan to use this environment as a development platform ?

For XFS, the development files would be the kernel ones, which I think it's completely out of scope.
Ceph runs in containers, so I'm not sure how useful would be to install the development packages.

Reply:
This particular ansible playbook was essentially copied from my notes. I needed a way to debug test failures and wanted a way to compile samba on the storage machines so that I could insert test code to understand the code flow. This ansible script essentially is an aid for me to quickly setup a build environment on the storage machine to test it. 

This is more of a developer aid(and that too mostly me) and probably should sit more in a wiki for this project.  We can get rid of this script as it will cause more confusion then help developers at this stage.

Reply:
I would then say we don't create a new one for CentOS 9 until we decide on what needs to be included.

Reply:
That's fine.